packages/@expo/cli/src/start/server/metro/MetroBundlerDevServer.ts

```diff
@@ -910,6 +932,13 @@ export class MetroBundlerDevServer extends BundlerDevServer {
     this.pendingRouteOperations.clear();
   }
 
+  // Metro HMR
+
+  private setupHmr(url: URL) {
+    debug('[CLI]: Register SSR HMR bundle URL:', url.toString());
+    // TODO: Pull in HMR SSR code.
```

Comment:
What needs to happen for HMR & SSR?

Reply:
A self-contained branch must be merged https://github.com/expo/expo/compare/%40evanbacon/cli/ssr-hmr-stub?expand=1